### PR TITLE
add require opa env

### PR DESCRIPTION
**Pull request by wurstbrot**
## Description

Add the check of an env variable.

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [x] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.

### CODE REVIEW

**Review by chipzoller — CHANGES_REQUESTED** [review]
Thanks for this, but please place this policy in the `other` folder and not `pod-security`.

**Comment by chipzoller** [comment]
We also require DCO sign-off on all contributions.

**Comment by chipzoller** [comment]
We have to approve for first-time contributors, but yes they do run automatically.

**Comment by wurstbrot** [comment]
> We have to approve for first-time contributors, but yes they do run automatically.

In the test logs, I do not see my tests getting executed for the policy "validate-env-vars". Do I need to add the integration somewhere?

File: other/require-opa-env/require-opa-enabled.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: validate-env-vars
+  annotations:
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/title: Check envirnoment variables
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/subject: Pod
+    kyverno.io/kubernetes-version: "1.22-1.23"
```
Comment (chipzoller) [inline other/require-opa-env/require-opa-enabled.yaml:10]: Can you also test on K8s 1.24 and/or 1.25? Update as needed.

**Review by chipzoller — CHANGES_REQUESTED** [review]

File: other/require-opa-env/require-opa-enabled.yaml (hunk 1 above)
Comment (wurstbrot) [reply other/require-opa-env/require-opa-enabled.yaml:10]: I am happy to test with kyverno-cli, but I am not sure how to test it with Kubernetes.

Comment (chipzoller) [reply other/require-opa-env/require-opa-enabled.yaml:10]: KinD? K3d? Minikube?

**Comment by chipzoller** [comment]
@wurstbrot will you please help to close this PR out?

**Comment by chipzoller** [comment]
Accepting as-is to fix in my fork.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**